seqr/utils/elasticsearch/constants.py

```diff
@@ -254,15 +253,17 @@
     for sort, sort_field in PREDICTOR_SORT_FIELDS.items()
 })
 
+CLINVAR_FIELD = 'clinvar'
 CLINVAR_FIELDS = ['clinical_significance', 'variation_id', 'allele_id', 'gold_stars']
+HGMD_FIELD = 'hgmd'
 HGMD_FIELDS = ['accession', 'class']
 GENOTYPES_FIELD_KEY = 'genotypes'
 HAS_ALT_FIELD_KEYS = ['samples_num_alt_1', 'samples_num_alt_2', 'samples']
 SORTED_TRANSCRIPTS_FIELD_KEY = 'sortedTranscriptConsequences'
 NESTED_FIELDS = {
     field_name: {field: {} for field in fields} for field_name, fields in {
-        'clinvar': CLINVAR_FIELDS,
-        'hgmd': HGMD_FIELDS,
+        CLINVAR_FIELD: CLINVAR_FIELDS,
```

Comment:
Since CLINVAR_FIELD and CLINVAR_FIELDS represent things that have different meanings, I'd use a different var name - maybe CLINVAR_KEY?
Same for HGMD.